File: JavaFragment/app/src/main/java/com/mhmmdyzci/javafragment/MainActivity.java
package com.mhmmdyzci.javafragment;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {
    FragmentManager fragmentManager;
    FragmentTransaction fragmentTransaction;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        fragmentManager = getSupportFragmentManager();
    }
    public void goToFirst(View view){

        fragmentTransaction = fragmentManager.beginTransaction();
        FirstFragment firstFragment = new FirstFragment();
        fragmentTransaction.replace(R.id.frame_layout, firstFragment).commit();


    }
    public void goToSecond(View view){
        fragmentTransaction = fragmentManager.beginTransaction();
        SecondFragment secondFragment = new SecondFragment();
       // fragmentTransaction.add(R.id.frame_layout, secondFragment).commit();
        //add yerine replace dersek fragmentle üst üste binmez
        fragmentTransaction.replace(R.id.frame_layout, secondFragment).commit();

    }
}

File: CoroutineKotlin/app/src/main/java/com/example/coroutinekotlin/MainActivity.kt
package com.example.coroutinekotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.coroutines.GlobalScope
import kotlinx.coroutines.launch

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
/*
        GlobalScope.launch {
            repeat(100000){
                launch {
                    println("android")
                }
            }
        }
      */
    }
}